packages/compass-e2e-tests/tests/atlas-login.test.ts

```diff
@@ -1,336 +0,0 @@
-import type { CompassBrowser } from '../helpers/compass-browser';
-import {
-  init,
-  cleanup,
-  screenshotIfFailed,
-  Selectors,
-  skipForWeb,
-  TEST_COMPASS_WEB,
-  DEFAULT_CONNECTION_NAME_1,
-} from '../helpers/compass';
-import type { Compass } from '../helpers/compass';
-import type { OIDCMockProviderConfig } from '@mongodb-js/oidc-mock-provider';
-import { OIDCMockProvider } from '@mongodb-js/oidc-mock-provider';
-import path from 'path';
-import { expect } from 'chai';
-import { createNumbersCollection } from '../helpers/insert-data';
-import { startMockAtlasServiceServer } from '../helpers/atlas-service';
-import type { Telemetry } from '../helpers/telemetry';
-import { startTelemetryServer } from '../helpers/telemetry';
-
-const DEFAULT_TOKEN_PAYLOAD = {
-  expires_in: 3600,
-  payload: {
-    groups: ['testgroup'],
-    sub: 'testuser',
-    aud: 'resource-server-audience-value',
-  },
-};
-
-function getTestBrowserShellCommand() {
-  return `${process.execPath} ${path.resolve(
-    __dirname,
-    '..',
-    'fixtures',
-    'curl.js'
-  )}`;
-}
-
-describe('Atlas Login', function () {
```

Comment:
I could also skip these if re-introducing is likely enough I assume we're not going to delete `stores/atlas-login`.

Reply:
I think you should clean it up, but I don't think you necessarily need to do it in this PR. This plugin was the first attempt at this sort of hight level cross plugin interface and implementation and so contains some patterns that we moved away from, it's probably for the better if we can completely drop it from the codebase

Reply:
added https://github.com/mongodb-js/compass/pull/7150 to handle the cleeanup